gatsby-config.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
   siteMetadata: {
+    // Replace the Site URL with your domain, ex. https://agency.gov 
+    siteUrl: 'https://agency.gov',
```

Comment:
Please change `agency` on lines 3 and 4 to `example` - I've had people get confused and actually put "agency.gov" in things before.

Reply:
Updated!